code/modules/modular_computers/file_system/programs/budgetordering.dm

```diff
@@ -145,15 +147,22 @@
 		data["cart"] += cart_list[item_id]
 
 	data["requests"] = list()
-	for(var/datum/supply_order/SO in SSshuttle.request_list)
+	for(var/datum/supply_order/order in SSshuttle.request_list)
+		amount_by_name[order.pack.name] += 1
```

Comment:
would it be valuable to do something like `var/datum/supply_pack/pack = order.pack` for some speed since we access the same three variables every time or is that too much nit?

Reply:
idk, sure